extra-libs/fftw/autobuild/build

```diff
@@ -2,15 +2,32 @@ cd "$SRCDIR"/fftw-${PKGVER/+/-}
 CONFIGURE="./configure F77=gfortran --prefix=/usr \
            --enable-shared --enable-threads \
            --enable-openmp"
+CMAKE_OPTIONS="
+	-Bcmake-gen
+	-DCMAKE_INSTALL_PREFIX=/usr
+	-DCMAKE_BUILD_TYPE=None
```

Comment:
Why?

Reply:
Cmake is required to generate `FFTW3LibraryDepends.cmake`